src/main/java/seedu/address/logic/parser/SellCommandParser.java

```diff
@@ -44,9 +54,17 @@ public SellCommand parse(String args) throws ParseException {
         Goods goods = ParserUtil.parseGoods(argMultimap.getValue(PREFIX_GOODS).orElse(""));
         Price price = ParserUtil.parsePrice(argMultimap.getValue(PREFIX_PRICE).orElse(""));
         Quantity quantity = ParserUtil.parseQuantity(argMultimap.getValue(PREFIX_QUANTITY).orElse(""));
+        Date date;
```

Comment:
I like how you used LocalDate to make sure transactions will have a date field if users do not input them manually